docs/csharp/toc.yml

```diff
@@ -156,7 +156,9 @@ items:
     href: whats-new/csharp-9.md
   - name: C# 8.0
     href: whats-new/csharp-8.md
-  - name: Compiler breaking changes
+  - name: Breaking changes since C# 10
```

Comment:
It's a little confusing to have two breaking changes articles here. Would this one fit better nested under the C# 11 article?

Reply:
That's a great idea. I didn't like the way it looked in the TOC, but I hadn't thought of a better presentation.